.github/workflows/test_package_openmpi.yml

```
@@ -0,0 +1,80 @@
+name: Test package with openmpi
+
+on:
+  push:
+    # The CI is executed on every push on every branch
+    branches:
+      - "**"
+  pull_request:
+    # The CI is executed on every pull request to the main branch
+    branches:
+      - main
+
+  workflow_call:
+  workflow_dispatch:
+  schedule:
+    # The CI is executed every day at 8am
+    - cron: "0 8 * * *"
+
+jobs:
+
+  create-datasets:
+    uses: ./.github/workflows/create_legacy_data.yml
+
+  create-legacy-datasets:
+    uses: ./.github/workflows/create_legacy_checkpoint.yml
+
+  test-code:
+    runs-on: "ubuntu-22.04"
+    needs: [create-datasets, create-legacy-datasets]
+    container: ghcr.io/fenics/test-env:current-openmpi
+    env:
+      DEB_PYTHON_INSTALL_LAYOUT: deb_system
+      PETSC_ARCH: "linux-gnu-real64-32"
+      OMPI_ALLOW_RUN_AS_ROOT: 1
+      OMPI_ALLOW_RUN_AS_ROOT_CONFIRM: 1
+
+    strategy:
+      matrix:
+        adios2: ["default", "v2.10.0-rc1"]
```

```suggestion
        adios2: ["default"] # , "v2.10.0-rc1"]
```
Deactivate v2.10.0-rc1 waiting for #60